src/app/src/Sidebar.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box, Heading } from 'rebass';
+
+const StyledSidebar = styled(Box)`
+    width: 300px;
+`;
+
+const Sidebar = () => {
+    return (
+        <StyledSidebar bg='lightgray'>
```

Comment:
Let's be consistent, using the variable defined style or the inline definitions but not both at the same time. Here can we move the `bg` tag to the `StyledSidebar` var?

Reply:
Okay! I did it because `lightgray` is a variable from `theme.js`—I'll try and see if there's some way to do this differently.

Reply:
I imagine you can use the theme variable in the templated style? 🤔

Reply:
https://www.styled-components.com/docs/advanced#theming

Reply:
Since we're built on Styled Systems, we get a function called `themeGet` that is less wordy than the one from styled-components. We do have to import the function from `styled-system` where we want it, however.